Spi intro (#10)
wix-incubator/corvid-spi-docs: guides/overview-ecom-shipping-rates.md

# eCommerce Shipping Rates (Beta)

> Provide custom or external shipping rate business logic that does not come natively with Wix into your eCommerce purchase flows. For example, you're not limited to how Wix business apps, such as Wix Stores and Wix Bookings, define the number of payments and shipping providers to choose from, and/or how payments and shipping rates are displayed on your site. 

Reviewer: ```suggestion
Provide custom or external shipping rate options that don't not come natively with Wix in your eCommerce purchase flows. You're not limited to how Wix business apps, such as Wix Stores and Wix Bookings, define the shipping providers to choose from, and/or how payments and shipping rates are displayed on your site. 
```

"and/or how payments and shipping rates are displayed on your site. "

Unless they changed things since I worked on this, you can't use the SPI to change how things are displayed on a site.

Author: @roysha-wix  Roy, can you let me know which is correct? Can you change the way the shipping rates are displayed?

Reviewer 2: Provide custom or external shipping rate options that don't come natively with Wix in your eCommerce purchase flows. For example, you're not limited to how Wix apps, such as Wix Stores, define the shipping providers to choose from.

Reviewer 2: you can't change the way they are displayed, and not sure why payments are also mentioned
